src/PrestaShopBundle/Form/Admin/Product/ProductSpecificPrice.php

```diff
@@ -241,11 +241,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             )
             ->add(
                 'sp_reduction',
-                FormType\MoneyType::class,
+                FormType\NumberType::class,
                 [
                     'label' => $this->translator->trans('Reduction', [], 'Admin.Catalog.Feature'),
                     'required' => false,
-                    'currency' => $this->currency->iso_code,
+                    'constraints' => [
+                        new Assert\Type(['type' => 'numeric']),
+                    ],
```

Comment:
I wonder if this code is redundant with the `NumberType` behavior.

Reply:
I will remove this code.